### PR TITLE
Fix RHEL7 rules sshd_use_strong_macs and sshd_use_strong_ciphers.

### PR DESCRIPTION
**Pull request by matejak**
- Implemented Bash remediations according to rule description.
- Synced sshd_use_strong_ciphers OVAL according with the rule description.

### CODE REVIEW

File: linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/bash/shared.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# platform = multi_platform_all
+
+{{{ bash_sshd_config_set(parameter="Ciphers", value="aes128-ctr,aes192-ctr,aes256-ctr,chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr") }}}
```
Comment (jan-cerny) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/bash/shared.sh]: aes128-ctr and aes256-ctr is twice in the list

File: linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/oval/shared.xml [hunk 2]
```diff
@@ -1 +1 @@
-{{{ oval_sshd_config(parameter="Ciphers", value="((chacha20-poly1305@openssh\.com|aes256-gcm@openssh\.com|aes128-gcm@openssh\.com|aes256-ctr|aes128-ctr),?)") }}}
+{{{ oval_sshd_config(parameter="Ciphers", value="((aes128-ctr|aes192-ctr|aes256-ctr|chacha20-poly1305@openssh\.com|aes256-gcm@openssh\.com|aes128-gcm@openssh\.com|aes256-ctr|aes128-ctr),?)") }}}
```
Comment (jan-cerny) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/oval/shared.xml]: aes128-ctr and aes256-ctr is twice in the list

Comment (jan-cerny) [reply linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/oval/shared.xml]: the rule.yml also has it twice and the OCIL there is missing aes192-ctr, could you please fix rule.yml as well?

Comment (matejak) [reply linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/oval/shared.xml]: And the OVAL has been missing + as well. I will indeed add some scenarios :-)

File: linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/rule.yml [hunk 3]
```diff
@@ -9,8 +9,7 @@ description: |-
     Counter (CTR) mode is also preferred over cipher-block chaining (CBC) mode.
     The following line in <tt>/etc/ssh/sshd_config</tt>
     demonstrates use of those ciphers:
-    <pre>Ciphers aes128-ctr,aes192-ctr,aes256-ctr</pre>
-    <pre>chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr</pre>
+    <pre>Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr</pre>
```
Comment (jan-cerny) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/rule.yml]: What about aes192-ctr?

Comment (matejak) [reply linux_os/guide/services/ssh/ssh_server/sshd_use_strong_ciphers/rule.yml]: Oh, right. Fixed by rebase.

**Pull request merged**